src/Commands/InvalidateCacheCommand.php

```diff
@@ -20,22 +20,31 @@ class InvalidateCacheCommand extends Command
 
     public $latestCheck;
 
+    public $staticCachePath;
+
     public function handle(Cacher $cacher, Writer $writer): void
     {
-        $this->latestCheck = $this->getLatestCheckDate();
-
-        if (!$this->latestCheck) {
-            $this->info('Cleared all urls (as we do not have a latest check date yet)');
-            $cacher->flush();
-            $this->setLatestCheckDate($writer);
-            return;
-        }
-        $this->setLatestCheckDate($writer);
-
         $stores = Rapidez::getStores();
+        $staticCachePathConfig = 'statamic.static_caching.strategies.full.path';
 
         foreach ($stores as $store) {
             Rapidez::setStore($store);
+           
+            if (is_array(config('statamic.static_caching.strategies.full.path'))) {
+                $staticCachePathConfig = $staticCachePathConfig . '.' . $store['code'];
+            }
+
+            $this->staticCachePath = config($staticCachePathConfig);
+
+            $this->latestCheck = $this->getLatestCheckDate();
+
+            if (!$this->latestCheck) {
+                $this->info('Cleared all urls (as we do not have a latest check date yet)');
+                $cacher->flush();
+                $this->setLatestCheckDate($writer);
+                return;
```

Comment:
Since we're within a foreach now, wouldn't we want to use continue in case other stores do have a latestcheck file?

Reply:
🔪